config.go

```diff
@@ -180,10 +178,16 @@ func init() {
 		},
 	}
 
-	appConfigPath = filepath.Join(AppPath, "conf", "app.conf")
+	AppPath, _ = filepath.Abs(filepath.Dir(os.Args[0]))
+	workPath, _ = os.Getwd()
```

Comment:
you can change to `workPath, err : = os.Getwd()`

remove the workPath declare. and also I think we need to check all the err. if has the err just panic at start the application.